validate-config/validate-config.yaml

```
@@ -0,0 +1,67 @@
+name: Hub Config Validation (R)
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: main
+    paths:
+      - 'hub-config/**'
+      - '!**README**'
+
+jobs:
+  validate-hub-config:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          install-r: false
+          use-public-rspm: true
+
+      - name: Update R
+        run: |
+          sudo apt-get update
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          packages: hubverse-org/hubAdmin, any::sessioninfo
+
+      - name: Run validations
+        id: validate
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          workspace <- "${{ github.workspace }}"
```

```suggestion
          hub_path <- "${{ github.workspace }}"
```
To be more intuitive?

---

In all honesty I was envisaging assigning this to an environment variable like `PR_NUMBER: ${{ github.event.number }}` is above that can be used throughout the step/job as necessary.

e.g is it required again in the comment on PR step or is that sth different?

---

Ahhhh that makes sense. I will go ahead and set it as an environment variable at the top of the job.

---

This way, when we port it over to https://github.com/hubverse-org/hubUtils/issues/158, we will simply append the workspace with the correct path.